### ISSUE TITLE
World suddenly unplayable

### ISSUE THREAD

**Issue by Onzeoo** [GTNewHorizons/GT-New-Horizons-Modpack#16714]
### Your GTNH Discord Username

@Onzeo

### Your Pack Version

2.6.1

### Your Server

SP

### Java Version

Java 8

### Type of Server

None

### Your Actions

Early steam age, was leaving base to go adventure a bit, game suddenly froze and became unresponsive - waited a while but didnt repair itself so i force closed. reopened and trying to play the same world immediately game me a error couldnt load message with "Unknown mod", tried again and world never loaded beyond 0%.
Copied instance in Multimc in hopes that'd fix it, did not as exact same thing happened.
Copied instance again and loaded a backup of 30 minutes before, same issue.
Actually the third time this has happened, first time on this world. have assigned 11gb of ram and have more than enough free disk space.

Only added mod is "Windowed fullscreen", 
Can provide save files

EDIT: Randomly fixed itself on copied instance #4 using the original save (at the exact spot it crashed), still unsure on what has been causing this problem for me

### Crash Report

[log.txt](https://github.com/user-attachments/files/16147984/log.txt)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [ ] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by Onzeoo** [comment]
Same issue has returned, heres the crash report for "The Following mods have been identified as potential causes: Unknown" https://mclo.gs/KNG5sl3

**Comment by Caedis** [comment]
Thats a different error btw

**Comment by Pxx500** [comment]
> Same issue has returned, heres the crash report for "The Following mods have been identified as potential causes: Unknown" https://mclo.gs/KNG5sl3

For now there is nothing that can be done to prevent this bug, it's new and as of now there is no known fix except deleting affected blocks.
For that you need a new hodgepodge, regenerate config (or just add needed lines which are in the link). The block deletion is turned off by default to show player that something is broken, to enable it do as instructed here: https://discord.com/channels/181078474394566657/1211702130092539965/1254843421651107903

**Comment by Caedis** [comment]
Yeah, we are scratching our head on the cause. Its possible the ores have been doing this the entire time but the corruption check we added recently started to catch it. Ores dont tick so there isnt much to break if it is "corrupted"

**Comment by Onzeoo** [comment]
> > Same issue has returned, heres the crash report for "The Following mods have been identified as potential causes: Unknown" https://mclo.gs/KNG5sl3
> 
> For now there is nothing that can be done to prevent this bug, it's new and as of now there is no known fix except deleting affected blocks. For that you need a new hodgepodge, regenerate config (or just add needed lines which are in the link). The block deletion is turned off by default to show player that something is broken, to enable it do as instructed here: https://discord.com/channels/181078474394566657/1211702130092539965/1254843421651107903

Im not exact sure where im supposed to put that config? just in the general config folder or a specific place? im new to this modpack and not familiar with the technical workings

**Comment by Onzeoo** [comment]
> > Same issue has returned, heres the crash report for "The Following mods have been identified as potential causes: Unknown" https://mclo.gs/KNG5sl3
> 
> For now there is nothing that can be done to prevent this bug, it's new and as of now there is no known fix except deleting affected blocks. For that you need a new hodgepodge, regenerate config (or just add needed lines which are in the link). The block deletion is turned off by default to show player that something is broken, to enable it do as instructed here: https://discord.com/channels/181078474394566657/1211702130092539965/1254843421651107903

I believe ive correctly followed the guide (added .Jar to mods folder/removed old one, changed config file) but its still occurring when im entering the same general area. ive attach the crash log i got after double checking i had properly applied the "Fix"
[crash-2024-07-14_21.10.48-server.txt](https://github.com/user-attachments/files/16228544/crash-2024-07-14_21.10.48-server.txt)
